packages/models/src/lib/plugin-config-runner.ts

```diff
@@ -1,5 +1,13 @@
 import { z } from 'zod';
 import { filePathSchema } from './implementation/schemas';
+import { auditOutputsSchema } from './plugin-process-output';
+
+export const outputTransformSchema = z
+  .function()
+  .args(z.array(z.record(z.string(), z.unknown())))
+  .returns(auditOutputsSchema);
```

Comment:
Maybe we can support async transformations? :thinking: 

```suggestion
  .returns(z.union([auditOutputsSchema, z.promise(auditOutputsSchema)]));
```

Reply:
❤️